ui/src/hooks/useExportToImage.ts

```diff
@@ -11,6 +11,17 @@ export const useExportToImage = () => {
   const context = useContext(MyContext);
   const selectedVolumeName = context.store.volume?.volumeName;
 
+  const navigateToImage = (imageName: string, imageId: string, tag: string) => {
+    ddClient.desktopUI.navigate
+      .viewImage(imageId, tag || "latest")
+      .catch(() => {
+        sendNotification(`Couldn't navigate to image ${imageName}`, {
+          name: "Try again",
+          onClick: () => navigateToImage(imageName, imageId, tag),
```

Comment:
The users need to dismiss the notification at some point, maybe they don't want to retry indefinitely :)

Reply:
added in 3974eda3be25ace549f1e4381c9e1c2f3803428e